client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -204,46 +189,25 @@ const ScreenMain = ( {
 							checked={ Boolean( selectedFooter ) }
 							icon={ footer }
 							aria-label={ translate( 'Footer' ) }
-							onClick={ () => onMainItemSelect( { name: 'footer', isPanel: true } ) }
+							onClick={ () => onMainItemSelect( 'footer' ) }
 							active={ getIsActiveButton( 'footer' ) }
 						>
 							{ translate( 'Footer' ) }
 						</NavigatorItem>
 					</NavigatorItemGroup>
-					<NavigatorItemGroup title={ translate( 'Styles' ) }>
-						<>
-							<NavigatorButtonAsItem
-								checked={ hasColor }
-								path={ NAVIGATOR_PATHS.COLOR_PALETTES }
-								icon={ color }
-								aria-label={ translate( 'Colors' ) }
-								onClick={ () => onMainItemSelect( { name: 'color-palettes' } ) }
-							>
-								{ translate( 'Colors' ) }
-							</NavigatorButtonAsItem>
-							<NavigatorButtonAsItem
-								checked={ hasFont }
-								path={ NAVIGATOR_PATHS.FONT_PAIRINGS }
-								icon={ typography }
-								aria-label={ translate( 'Fonts' ) }
-								onClick={ () => onMainItemSelect( { name: 'font-pairings' } ) }
-							>
-								{ translate( 'Fonts' ) }
-							</NavigatorButtonAsItem>
-						</>
-					</NavigatorItemGroup>
 				</VStack>
 				{ ! surveyDismissed && <Survey setSurveyDismissed={ setSurveyDismissed } /> }
 			</div>
 			<div className="screen-container__footer">
 				<Button
 					className="pattern-assembler__button"
-					primary
-					aria-disabled={ disabled }
-					onMouseDown={ handleMouseDown }
+					aria-disabled={ isButtonDisabled }
 					onClick={ handleClick }
+					label="Add your first pattern to get started."
+					showTooltip={ isButtonDisabled }
+					variant="primary"
 				>
-					{ translate( 'Save and continue' ) }
+					{ translate( 'Pick your style' ) }
```

Comment:
Seems that this is the only new translation in this PR? If that's the case, we can use `hasTranslation()` or `useHasEnTranslation()` in order to merge this without waiting for translation. We can keep using "Save and continue" in the meantime.

Reply:
Oh, I just found out that there's also this heading:
![Screenshot 2023-08-23 at 2 18 57 PM](https://github.com/Automattic/wp-calypso/assets/797888/10335c26-8d3b-4d2f-ac63-1a8ac7b9d931)

I think we can use Style for this, WDYT?

Reply:
Okay, let's use `hasTranslation` and use "Save and continue" and "Styles" in the meantime. Thanks

Reply:
✅ Done